python/nvstrings/nvstrings.py

```diff
@@ -1648,97 +1648,6 @@ def rstrip(self, to_strip=None):
             rtn = nvstrings(rtn)
         return rtn
 
-    def lower(self):
-        """
-        Convert each string to lowercase.
-        This only applies to ASCII characters at this time.
-
-        Examples
-        --------
-        >>> import nvstrings
-        >>> s = nvstrings.to_device(["Hello, Friend","Goodbye, Friend"])
-        >>> print(s.lower())
-        ['hello, friend', 'goodbye, friend']
-
-        """
-        rtn = pyniNVStrings.n_lower(self.m_cptr)
-        if rtn is not None:
-            rtn = nvstrings(rtn)
-        return rtn
-
-    def upper(self):
-        """
-        Convert each string to uppercase.
-        This only applies to ASCII characters at this time.
-
-        Examples
-        --------
-        >>> import nvstrings
-        >>> s = nvstrings.to_device(["Hello, friend","Goodbye, friend"])
-        >>> print(s.upper())
-        ['HELLO, FRIEND', 'GOODBYE, FRIEND']
-
-        """
-        rtn = pyniNVStrings.n_upper(self.m_cptr)
-        if rtn is not None:
-            rtn = nvstrings(rtn)
-        return rtn
-
-    def capitalize(self):
-        """
-        Capitalize first character of each string.
-        This only applies to ASCII characters at this time.
-
-        Examples
-        --------
-        >>> import nvstrings
-        >>> s = nvstrings.to_device(["hello, friend","goodbye, friend"])
-        >>> print(s.capitalize())
-        ['Hello, friend", "Goodbye, friend"]
-
-        """
-        rtn = pyniNVStrings.n_capitalize(self.m_cptr)
-        if rtn is not None:
-            rtn = nvstrings(rtn)
-        return rtn
-
-    def swapcase(self):
-        """
-        Change each lowercase character to uppercase and vice versa.
-        This only applies to ASCII characters at this time.
-
-        Examples
-        --------
-        >>> import nvstrings
-        >>> s = nvstrings.to_device(["Hello, Friend","Goodbye, Friend"])
-        >>> print(s.swapcase())
-        ['hELLO, fRIEND', 'gOODBYE, fRIEND']
-
-        """
-        rtn = pyniNVStrings.n_swapcase(self.m_cptr)
-        if rtn is not None:
-            rtn = nvstrings(rtn)
-        return rtn
-
-    def title(self):
-        """
-        Uppercase the first letter of each letter after a space
-        and lowercase the rest.
-        This only applies to ASCII characters at this time.
-
-        Examples
-        --------
-        >>> import nvstrings
-        >>> s = nvstrings.to_device(["Hello friend","goodnight moon"])
-        >>> print(s.title())
-        ['Hello Friend', 'Goodnight Moon']
-
-        """
-        rtn = pyniNVStrings.n_title(self.m_cptr)
-        if rtn is not None:
-            rtn = nvstrings(rtn)
-        return rtn
```

Comment:
Don't remove these from nvstrings yet, someone may be using these directly from nvstrings still.

Reply:
Ok. But let the recod show that `StringMethods` falls back to these, so if not implemented properly, the existing tests may still pass.

Reply:
Yea it falls back to them only if there isn't a function implemented in the `StringMethods` class for it to use. So if someone goes outside of our expected API that's perfectly fine for it to fallback and hit the `NVStrings` implementation for the time being.